python/pyspark/sql/functions.py

```diff
@@ -2042,7 +2042,7 @@ def trunc(date, format):
     ----------
     date : :class:`~pyspark.sql.Column` or str
     format : str
-        'year', 'yyyy', 'yy' or 'month', 'mon', 'mm'
+        'year', 'yyyy', 'yy' or 'month', 'mon', 'mm', or 'week' or 'quarter'
```

Comment:
```suggestion
        'year', 'yyyy', 'yy', 'month', 'mon', 'mm', 'week' or 'quarter'
```

Reply:
Actually, can we copy the docs in from Scala's trunc? 

also should fix https://github.com/apache/spark/blob/b5ee6d008d9cc3ad5061ad7415f7099e27769853/R/pkg/R/functions.R#L49-L51 too

Reply:
I took the Scala doc